wiki/redirect.yaml

```diff
@@ -877,6 +877,11 @@
 "language_mod":                    "People/The_Team/Global_Moderation_Team#grouped-by-languages-moderated"
 "lm":                              "People/The_Team/Global_Moderation_Team#grouped-by-languages-moderated"
 
+"lsd":                             "People/The_Team/Language_Surveillance_Division"
+"chatops":                         "People/The_Team/Language_Surveillance_Division"
+"chat_ops":                        "People/The_Team/Language_Surveillance_Division"
+"chat_moderators":                 "People/The_Team/Language_Surveillance_Division"
```

Comment:
Are these required / used anywhere? Seems weird to have them linking to a defunct team rather than the GMT?

Reply:
absolutely not. will redirect them to the GMT instead